docs/ConvertDataOperation.md

```diff
@@ -0,0 +1,194 @@
+# Convert Data Operation
+The ```$convert-data``` operation allows data conversion from legacy formats (currently supports **Hl7v2** only) to FHIR format. This feature is currently turned off by default. To enable the feature, update the `FhirServer:Operations:ConvertData:Enabled` setting to be true.
+
+## Conversion Template Collection
+The convert data operation is based on the [FHIR Converter Project](https://github.com/microsoft/FHIR-Converter/tree/dotliquid).
+To make a data conversion call, you need a template collection to define the mappings between different data formats.
+
+We have a built-in template collection (hl7v2 to FHIR R4) which is a copy from the FHIR Converter project.
+You can directly use the built-in template collection reference ```microsofthealth/fhirconverter:default``` in the request payload.
+
+If you need to customize the templates, you can get a copy from the FHIR Converter project and customize it as you wish, then upload it to your ACR (Azure Container Registry). In this case, the template collection reference should be ```<RegistryServer>/<imageName>:<imageTag>```. Both image tags and digests are supported in the reference string.
+
+In order to integrate the ACR to your FHIR instance, you need update `FhirServer:Operations:ConvertData:ContainerRegistries:0:RegistryServer` to your ACR login server. Besides, you need to give your App service the ```AcrPull``` permission for the ACR of your choice.
+
+## Request format
+The request payload is passed as [Parameters](http://hl7.org/fhir/parameters.html) resource to the convert data api.
```

Comment:
wherever we are using 'convert data api', consider changing to $convert-data api.

Reply:
Fixed, thanks